### PR TITLE
Create: Product card created

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
create: Made vertical product card component

Screenshot_2022-02-16_13-38-56

preview link : https://deploy-preview-23--fabelleui.netlify.app/documentation/docs

### CODE REVIEW

**Review by Rockyraven — APPROVED** [review]
All goods to me go ahead

File: documentation/docs.html [hunk 1]
```diff
@@ -475,8 +525,7 @@ <h3 class="small-title">Grid</h3>
                 </div>
             </div>
             <div class="component-code">
-                <iframe
-                    src="https://carbon.now.sh/embed/sTKpG2Pt6XPRbCvwjsue"
+                <iframe src="https://carbon.now.sh/embed/sTKpG2Pt6XPRbCvwjsue"
                     style="height: 372px; width: 100%; border:0; transform: scale(1); overflow:hidden; background: #090300;"
```
Comment (VLeads) [inline documentation/docs.html:529]: don't use inline style.

**Review by VLeads — APPROVED** [review]
Rest al code looks good!

**Review by Shubhamkr2610 — APPROVED** [review]
Looks good to me . You are good to go

**Pull request merged**